### PR TITLE
Re-register SMT client after changing namespace (bsc#1034676)

### PR DESCRIPTION
**Pull request by tbazant**
### Description
This update describes steps to change the namespace of an SMT client

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [x] To maintenance/SLE12SP4
- [x] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/smt_client.xml [hunk 1]
```diff
@@ -592,12 +592,31 @@
...
-   the production environment, modify <filename>/etc/SUSEConnect</filename> on
-   the client machine by setting:
+   To register a client in the testing environment, follow these steps:
   </para>
 
+  <procedure>
+   <step>
+    <para>
+     Delete a registration of the client host. Find more information in
+     <xref linkend="smt-registration-delete"/>
```
Comment (digitaltom) [inline xml/smt_client.xml:602]: This deletes the client on the server, but doesn't change the client itself I think. I would recommend to run `SUSEConnect --de-register` on the client.

Comment (tbazant) [inline xml/smt_client.xml:602]:
```suggestion
     De-register the client from the &smt; server by running <command>SUSEConnect --de-register</command> on the client host.
```

File: xml/smt_mirroring.xml [hunk 2]
```diff
@@ -570,11 +570,31 @@ repo/testing/SUSE/Updates/SLE-SERVER/12/x86_64/update/
...
-   To register a client in the testing environment, modify
-   <filename>/etc/SUSEConnect</filename> on the client machine as follows:
+   To register a client in the testing environment, follow these steps:
   </para>
 
+  <procedure>
+   <step>
+    <para>
+     Delete a registration of the client host. Find more information in
+     <xref linkend="smt-registration-delete"/>
```
Comment (tbazant) [inline xml/smt_mirroring.xml:580]:
```suggestion
     De-register the client from the &smt; server by running <command>SUSEConnect --de-register</command> on the client host.
```

File: xml/smt_mirroring.xml [hunk 3]
```diff
@@ -783,10 +803,31 @@ sudo chmod go-rwx -R /var/lib/smt/.gnupg</screen>
...
-      <filename>/etc/SUSEConnect</filename> file on the client machine:
+      To register a client in the testing environment, follow these steps:
      </para>
-<screen>namespace: testing</screen>
+
+     <procedure>
+      <step>
+       <para>
+        Delete a registration of the client host. Find more information in
+        <xref linkend="smt-registration-delete"/>
```
Comment (tbazant) [inline xml/smt_mirroring.xml:813]:
```suggestion
        De-register the client from the &smt; server by running <command>SUSEConnect --de-register</command> on the client host.
```

File: xml/smt_client.xml (hunk 1 above)
Comment (tbazant) [reply xml/smt_client.xml:602]: thanks, updated

**Pull request merged**